resources/java_memory_assistant/bin/clean-up.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -e
```

Comment:
GoLang it is then. You guys do not happen to have a build service for GoLang projects we could use for this? Otherwise I can set up something internally, but it is going to take time and it might be preferable to ship a first version with the ugly Shell scripts.